LayoutTests/fast/scrolling/mac/scrollbars/scrollbar-color.html

```diff
@@ -0,0 +1,76 @@
+<!DOCTYPE html> <!-- webkit-test-runner [ MockScrollbarsEnabled=false AsyncOverflowScrollingEnabled=true ] -->
+<html>
+<head>
+    <style>
+        html {
+            scrollbar-color: yellow blue;
+        }
+        body {
+            height: 1000px;
+        }
+        .scroller {
+            margin: 10px;
+            width: 200px;
+            height: 200px;
+            border: 1px solid black;
+            overflow: auto;
+            scrollbar-color: yellow blue;
+        }
+        .contents {
+            width: 100%;
+            height: 200%;
+        }
+    </style>
+    <script src="../../../../resources/js-test-pre.js"></script>
```

Comment:
nit: please use js-test.js instead of pre/post

Reply:
Fixed.